packages/reporter/src/hooks/hooks.tsx

```diff
@@ -59,14 +59,16 @@ const StudioNoCommands = () => (
 export interface HookProps {
   model: HookModel
   showNumber: boolean
+  testType?: Cypress.CypressSpecType
 }
 
-const Hook = observer(({ model, showNumber }: HookProps) => (
+const Hook = observer(({ model, showNumber, testType }: HookProps) => (
   <li className={cs('hook-item', { 'hook-failed': model.failed, 'hook-studio': model.isStudio })}>
     <Collapsible
       header={<HookHeader model={model} number={showNumber ? model.hookNumber : undefined} />}
       headerClass='hook-header'
-      headerExtras={model.invocationDetails && <HookOpenInIDE invocationDetails={model.invocationDetails} />}
+      // Temporarily removed from CT since it doesn't work. Will be re-enabled in https://github.com/cypress-io/cypress/issues/24549
+      headerExtras={model.invocationDetails && testType !== 'component' && <HookOpenInIDE invocationDetails={model.invocationDetails} />}
```

Comment:
Could we have used the Cypress here and not have had to change so many files?
```suggestion
      headerExtras={model.invocationDetails && Cypress.testingType !== 'component' && <HookOpenInIDE invocationDetails={model.invocationDetails} />}
```

Reply:
Nice! I didn't realize we had a global with that info